position1: 'qr1'
coordinates1: [0.0345450807056, 0.030581485681, 0.0065749829279, 0.999978384566]
position2: 'qr2'
coordinates2: [4.74468153877, -0.0978976867686, -0.0134701569269, 0.999909273321]
position3: 'qr3'
coordinates3: [2.60146477111, 1.81576926969, 0.729837905405, 0.683620239486]
position4: 'qr4'
coordinates4: [2.42130120034, -1.4149224531, -0.711948420065, 0.702231761719]